client/app/components/Board.tsx

```diff
@@ -0,0 +1,31 @@
+import React, {Component, ReactComponentElement} from 'react';
+
+import Card from './Card';
+import CardModel from '../data/card.model';
+
+export default class Board extends Component<any> {
+
+  public render(): ReactComponentElement<any> {
+    const cards: CardModel[] = this.props.cards;
+
+    console.log("cards", cards);
```

Comment:
Fun for this stage but just want to check you intended to leave it in

Reply:
nope